[2022-06-16 10:32:34,371] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: analytics.getdata scheduled__2022-05-25T00:00:00+00:00 [queued]>
[2022-06-16 10:32:34,605] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: analytics.getdata scheduled__2022-05-25T00:00:00+00:00 [queued]>
[2022-06-16 10:32:34,621] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-16 10:32:34,623] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-16 10:32:34,630] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-16 10:32:35,012] {taskinstance.py:1377} INFO - Executing <Task(MySqlOperator): getdata> on 2022-05-25 00:00:00+00:00
[2022-06-16 10:32:35,033] {standard_task_runner.py:52} INFO - Started process 3597 to run task
[2022-06-16 10:32:35,107] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'analytics', 'getdata', 'scheduled__2022-05-25T00:00:00+00:00', '--job-id', '445', '--raw', '--subdir', 'DAGS_FOLDER/testing_etl.py', '--cfg-path', '/tmp/tmpnc7k1i3z', '--error-file', '/tmp/tmpf3lwtmjg']
[2022-06-16 10:32:35,147] {standard_task_runner.py:80} INFO - Job 445: Subtask getdata
[2022-06-16 10:32:35,767] {task_command.py:370} INFO - Running <TaskInstance: analytics.getdata scheduled__2022-05-25T00:00:00+00:00 [running]> on host fe209149cfc1
[2022-06-16 10:32:37,109] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=analytics
AIRFLOW_CTX_TASK_ID=getdata
AIRFLOW_CTX_EXECUTION_DATE=2022-05-25T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-25T00:00:00+00:00
[2022-06-16 10:32:37,118] {mysql.py:82} INFO - Executing: SELECT * from Customer_T;
[2022-06-16 10:32:37,312] {base.py:68} INFO - Using connection ID 'mysql_con' for task execution.
[2022-06-16 10:32:37,411] {dbapi.py:213} INFO - Running statement: SELECT * from Customer_T;, parameters: None
[2022-06-16 10:32:37,423] {dbapi.py:221} INFO - Rows affected: 19
[2022-06-16 10:32:37,585] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=analytics, task_id=getdata, execution_date=20220525T000000, start_date=20220616T103234, end_date=20220616T103237
[2022-06-16 10:32:37,884] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-16 10:32:38,500] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
